metrics/status.global/global.go

```diff
@@ -20,10 +20,10 @@ import (
 )
 
 /*
-	Begin the collector with a const block.
-	First define DOMAIN.
-	Then define any OPT_* const.
-	Then define other const as needed.
+Begin the collector with a const block.
+First define DOMAIN.
+Then define any OPT_* const.
+Then define other const as needed.
```

Comment:
Did basic go fmt change this, or one of our Block git hooks?

Reply:
Yes, it was go fmt change but from go 1.20 which I had set as the version in my goland. I have changed it back to 1.18 and it's not reformatting it anymore. So it's a future go fmt change.

Saying that, I think we should use hermit to mange and pin go version in blip and also upgrade to 1.20. I will do that in a seperate PR. I am going leave this change maybe as it's going to be reformatted like this in future version of go fmt anyway?

Reply:
Don't use Hermit on this or any of our open source projects. Hermit is a Cash thing (not even a Square thing). I know Hermit is also open source, but an open source project should stick to go mod and nothing else. If users want Hermit or something else, they wrap the open source project in one of their own internal projects.

Reply:
I like `hermit` (or any tool for pinning tool versions), so you won't have to manually handle versions in different environments (dev, CI etc).

The problem with not using any such tool is exactly what happened here, everyone will have to explicitly remember to use the correct compiler every time they want to do something (IDE, cli). I wish go toolchain provided a way to enforce versions (will be hard to do in a sane way due to backward compatibility and all).